lib/perl/Genome/Model/SomaticValidation/Command/AlignmentStatsSummary.pm

```diff
@@ -143,7 +143,13 @@ sub _alignment_metrics_from_result {
     for my $lane (@per_lane_alignments) {
         $total_bases += $lane->total_base_count;
         $total_mapped_bases += $lane->total_aligned_base_count;
-        $mismatches += $lane->total_base_count * $lane->instrument_data->filt_error_rate_avg / 100;
+        my $filt_error_rate_avg;
+        if ($mismatches ne "NA" and $lane->instrument_data->can("filt_error_rate_avg")) {
```

Comment:
I believe this is intentional, but I just wanted to make sure I understood. If any one lane "cannot" `filt_error_rate_avg` then the value of both `mismatches` and `error_rate` for the whole report will be `NA` correct?